gui/default/index.html

```diff
@@ -369,6 +369,10 @@ <h4 class="panel-title">
                       <th><span class="fa fa-fw fa-refresh"></span>&nbsp;<span translate>Rescan Interval</span></th>
                       <td class="text-right">{{folder.rescanIntervalS}} s</td>
                     </tr>
+                    <tr ng-if="folder.longRescanIntervalS != 3600">
```

Comment:
I don't think we should expose this here, especially not with a cryptic camelcased name. See also below about the settings modal.